numpy/core/src/multiarray/_multiarray_tests.c.src

```diff
@@ -601,14 +601,14 @@ static PyObject *
 fromstring_null_term_c_api(PyObject *dummy, PyObject *byte_obj)
 {
     char *string;
+    PyArray_Descr *descr;
 
     string = PyBytes_AsString(byte_obj);
     if (string == NULL) {
         return NULL;
     }
-
-    return PyArray_FromString(
-                    string, -1, PyArray_DescrFromType(NPY_FLOAT64), -1, " ");
```

Comment:
What's the premise behind splitting this line?

Reply:
FWIW, I prefer one function per line when debugging, it saves stepping in to the wrong function and my mind can parse it more easily. But no big deal either way. If this is a blocker I will reformat it.

Reply:
That's reasonable, especially since usually multiple python API calls on a line means you forgot to check for NULL.